spec/services/reporting/total_user_count_report_spec.rb

```diff
@@ -70,4 +70,30 @@
       it_behaves_like 'a report with that user counted'
     end
   end
+
+  describe '#total_verified_user_report' do
+    let(:expected_count) { 0 }
+    let(:expected_report) do
+      [['Total verified users'], [expected_count]]
+    end
+
+    context 'with only a non-IdV user' do
+      let!(:unverified_user) { create(:user) }
+      let(:expected_count) { 0 }
```

Comment:
This and the one one 75 are technically redundant. I feel strongly that this one should be here to make the meaning  explicit. It also felt reasonable to have a default on line 75, but maybe I should remove that one?

Reply:
yeah i'd remove the default one if it's not needed